xradar/io/backends/cfradial1.py

```diff
@@ -327,22 +327,26 @@ def open_cfradial1_datatree(filename_or_obj, **kwargs):
         Import optional mandatory data and metadata, defaults to ``True``.
     site_coords : bool
         Attach radar site-coordinates to Dataset, defaults to ``True``.
+    engine: str
```

Comment:
I think this should work for now. No immediate need for other types, like `BackendEntrypoint`. :+1: